app/views/layouts/application.html.erb

```diff
@@ -177,15 +176,15 @@
         <div class="grid-container">
           <div class="usa-footer__logo grid-row grid-gap-2">
             <div class="grid-col">
-              <%= image_tag 'img/gsa-logo-rev.svg', alt: 'U.S. General Services Administration logo', class: 'usa-footer__logo-img usa-footer__slim usa-footer__medium' %>
+              <%= image_tag asset_url('gsa-logo-rev.svg'), alt: 'U.S. General Services Administration logo', class: 'usa-footer__logo-img usa-footer__slim usa-footer__medium' %>
             </div>
 
           </div>
         </div>
       </div>
     </footer>
 
-    <%= javascript_include_tag 'js/main.js' %>
-    <%= javascript_include_tag "application" %>
+    <%= javascript_include_tag asset_url('main.js') %>
```

Comment:
I wouldn't expect you'd have to use `asset_url` in any of the code changes, was it necessary?

Reply:
it definitely wasn't initially working without it, but a lot of other things have changed, so maybe it's possible now. i'll see!

Reply:
updated in https://github.com/18F/identity-dashboard/commit/594bc2dafc173305e4c2b377dcf9ad97baabc8e9 seems to work!